api/app/Providers/RouteServiceProvider.php

```diff
@@ -57,7 +57,21 @@ public function boot()
     protected function configureRateLimiting()
     {
         RateLimiter::for('api', function (Request $request) {
-            return Limit::perMinute(60)->by(optional($request->user())->id ?: $request->ip());
+            return Limit::perMinute(config('app.rate_limit'))->by(optional($request->user())->id ?: $request->ip())->response(function () {
+                return response([
+                    'message' => 'Rate Limit Reached for API'
+                ], 429);
+            });
+        });
+        RateLimiter::for('web', function (Request $request) {
+            return Limit::perMinute(config('app.rate_limit'))->by(optional($request->user())->id ?: $request->ip())->response(function () {
+                return response([
+                    'message' => 'Rate Limit Reached for Web'
+                ], 429);
+            });
+        });
+        RateLimiter::for('graphql', function (Request $request) {
```

Comment:
Suggest a comment here to clarify that this limiter is for the throttle directive which can be set independently of the of the route-based limiters and can raise graphql style error messages.